data_scripts/parseGenesToElastic.js

```diff
@@ -3,11 +3,11 @@ var level = require('level')
 var elasticsearch = require('elasticsearch')
 
 var client = new elasticsearch.Client({
-    host: 'localhost:9200',
```

Comment:
this *might break current GN

Reply:
Should I change it back? Without this change docker won't work

Reply:
I will test this locally and get back to you